src/replica/replica_stub.cpp

```diff
@@ -2901,7 +2895,7 @@ uint64_t replica_stub::gc_tcmalloc_memory(bool release_all)
             release_bytes -= 1024 * 1024;
         }
     }
-    _counter_tcmalloc_release_memory_size->set(tcmalloc_released_bytes);
+    METRIC_VAR_SET(tcmalloc_released_bytes, tcmalloc_released_bytes);
```

Comment:
@empiredan As we discuss offline, `tcmalloc_released_bytes` is better to be a counter type, we can improve it in next patch.

Reply:
OK, I'll fix this in the next PR.